vitest.config.ts

```diff
@@ -1,6 +1,8 @@
 /// <reference types="vitest" />
 import { configDefaults, defineConfig } from 'vitest/config';
 
+process.env.NEXT_PUBLIC_LANG = `en`;
```

Comment:
What is this for?

Reply:
the podcast spec file i introduced imports the lib/env.ts file, which throws if there is no lang defined. so this just makes it so that when we're running tests we always have a lang. we had a similar setup step in the old jest setup code as well.